### DISCUSSION TITLE
panic: runtime error: index out of range [0] with length 0

### DISCUSSION THREAD

**Discussion by Lowrence-Kalyansingh** [community/community#46834] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

when I run Hugo server (on Git bash) , after say 1/2 hour or so, I am getting this error :

panic: runtime error: index out of range [0] with length 0..
![image](https://user-images.githubusercontent.com/68060728/217787253-5c214243-5c4d-40f1-80ff-ad21f0c2360e.png)

can anyone help me.. what is the workaround for me please?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Lowrence-Kalyansingh ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
